NEWS.md

```diff
@@ -7,6 +7,7 @@
 * Fixing bug so that plot_time_series() works with upperbond_pad and expected_pad being all NA
 * Added notification about negative values in age.
 * Improvements in the documentation of `build_signals_table()`, `prepare_signals_agg_table()`
+* Bug fix inside `preprocess_data()` so that the tool works again when the age variable is not included in the data.
```

Comment:
Can you put this on top of SignalDetectionTool 0.3.0 so that it is clear that it belongs to the next version?

Reply:
and just pull develop and merge in your branch to resolve conflicts as there was one update of develop in the meantime ;)

Reply:
Sure! I'll let you know when these are done.